### PR TITLE
FormatURL handling of URLs with commas

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
As a comma is a valid character within parts of the URL the formatter was updated to accept it and the separator for multiple inputs has been switched to pipe ("|").

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/CommonScripts/Scripts/FormatURL/README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Strips, unquotes and unescapes URLs. If the URL is a Proofpoint or ATP URL, extracts its redirect URL.
+If more than one URL is passed to the formatter the separator must be a pipe ("|")
```
Comment (dantavori) [inline Packs/CommonScripts/Scripts/FormatURL/README.md:2]:
```suggestion
If more than one URL is passed to the formatter, the separator must be a pipe ("|").
```
let's add this to the script "comment" field as well

File: Packs/CommonScripts/Scripts/FormatURL/FormatURL.py [hunk 2]
```diff
@@ -34,7 +34,10 @@ class URLCheck(object):
     This class will build and validate a URL based on "URL Living Standard" (https://url.spec.whatwg.org)
     """
     sub_delims = ("!", "$", "&", "'", "(", ")", "*", "+", ",", ";", "=")
-    brackets = ("\"", "'", "[", "]", "{", "}", "(", ")", ",")
+    brackets = ("\"", "'", "[", "]", "{", "}", "(", ")")
+    url_code_points = ("!", "$", "&", "\"", "(", ")", "*", "+", ",", "-", ".", "/", ":", ";", "=", "?", "@",
+                            "_", "~")
```
Comment (dantavori) [inline Packs/CommonScripts/Scripts/FormatURL/FormatURL.py:39]: what is the purpose of this change?

Comment (Ni-Knight) [reply Packs/CommonScripts/Scripts/FormatURL/FormatURL.py:39]: I removed the comma from the `brackets` as it isnt a bracket and caused issues with the flow of the script. The comma is allowed and it is in the `url_code_points` but not as a bracket.

File: Packs/CommonScripts/Scripts/FormatURL/README.md (hunk 1 above)
Comment (Ni-Knight) [reply Packs/CommonScripts/Scripts/FormatURL/README.md]: Also added to the comment.

File: Packs/CommonScripts/Scripts/FormatURL/FormatURL.py (hunk 2 above)
Comment (dantavori) [reply Packs/CommonScripts/Scripts/FormatURL/FormatURL.py:39]: there is a definition of `url_code_points` also in `check_codepoint_validity()`, meaning what you defined will not be used there

Comment (Ni-Knight) [reply Packs/CommonScripts/Scripts/FormatURL/FormatURL.py]: Oh, wait, what, how did that even get there! I just deleted the comma from brackets and somehow must have pasted it there by mistake.

**Review by dantavori — APPROVED** [review]
